package.json

```diff
@@ -4736,48 +4736,6 @@
       {
         "language": "csharp",
         "scopes": {
-          "namespace": [
-            "entity.name.type.namespace.cs"
-          ],
-          "type": [
-            "entity.name.type"
-          ],
-          "class": [
-            "entity.name.type.class.cs"
-          ],
-          "enum": [
-            "entity.name.type.enum.cs"
-          ],
-          "interface": [
-            "entity.name.type.interface.cs"
-          ],
-          "struct": [
-            "entity.name.type.struct.cs"
-          ],
-          "typeParameter": [
```

Comment:
`typeParameter doesn't list a fallback in the table, we probably need to keep this one

Reply:
This one is interesting, in 1.x it maps as **entity.name.type.parameter**
![image](https://github.com/dotnet/vscode-csharp/assets/15952551/2e51f404-5ce6-4b65-b63b-3722b6898154)

Putting it back in the scopes list for 2.x it comes out as expected **entity.name.type.type-parameter.cs**, but should it be the more generic one or the .cs specific one?
![image](https://github.com/dotnet/vscode-csharp/assets/15952551/a52b9e03-6d63-49eb-a151-248be4d90d07)

Reply:
Actually ignore that I can see in the VS theme's it's using the full .cs version.

Reply:
I think we should set it to "entity.name.type.type-parameter" in the scopes

Reply:
> This one is interesting, in 1.x it maps as entity.name.type.parameter

Interesting - I don't know where that is coming from.  

> I think we should set it to "entity.name.type.type-parameter" in the scopes

Yup, I'm good with that.  If though `entity.name.type.parameter` looks better on other themes, and it works with the VS theme, I'm also OK with setting to that.  But `entity.name.type.type-parameter` seems to be what O# used.

Reply:
I checked this one on a couple of themes and it looks ok to me.